shenyu-common/src/main/java/org/apache/shenyu/common/dto/convert/selector/GrpcUpstream.java

```diff
@@ -45,6 +45,7 @@ private GrpcUpstream(final Builder builder) {
         this.weight = builder.weight;
         setStatus(statusValue);
         setTimestamp(builder.timestamp);
+        setNamespaceId(builder.namespaceId);
```

Comment:
why change this? need to change this?@ywj1352

Reply:
![image](https://github.com/user-attachments/assets/13f4c9be-319b-4a64-8407-629f103d0635)
From the functional point of view of the code does not need this at the moment.   But it's not wrong from an integrity point of view